### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by curtiskunkle**

### CODE REVIEW

File: dnddiceroller/DiceFileReader.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	} catch (FileNotFoundException ex) {
+	  Logger.getLogger(DiceFileReader.class.getName()).log(Level.SEVERE, null, ex);
+	}
+  }
+  
+  public List<String> getLines(){
+	return this.lines;
+  }
+  
+  public void read(){
```
Comment (ninjapanzer) [inline dnddiceroller/DiceFileReader.java:51]: Does this method have a singular responsibility?

It looks like it is reading and rolling

Comment (curtiskunkle) [inline dnddiceroller/DiceFileReader.java:51]: I see what you mean.... What if  the read method returned an array list of dice?

Comment (ninjapanzer) [inline dnddiceroller/DiceFileReader.java:51]: That sounds like a good plan

Comment (curtiskunkle) [inline dnddiceroller/DiceFileReader.java:51]: IMO roll should be the responsibility of DiceTower... but the only way that I can think of to make that work with the way I have it set up is to pass an arrayList of arrayLists to DiceTower and iterate through it in a method of DiceTower. I'm not sure if that's a real thing, and I couldn't get that to work so far.

Comment (froschs) [inline dnddiceroller/DiceFileReader.java:51]: Scope has been a pain for me with this project. That's why I had DiceReader doing so much work too.

**Pull request closed without merging**